system/lib/embind/bind.cpp

```diff
@@ -134,13 +134,6 @@ EMSCRIPTEN_BINDINGS(builtin) {
   register_integer<unsigned short>("unsigned short");
   register_integer<signed int>("int");
   register_integer<unsigned int>("unsigned int");
-#if __wasm64__
```

Comment:
Why are these removed? Bindings for long will no longer work. 

Somewhat of an aside: I have had it on my todo to get rid of the `register_bigint<int64_t>("int64_t");` and unsigned version since that is actually not a "primitive" type. They're really `long long int` and `unsigned long long int`

Reply:
Yeah locally it worked regardless for some reason, but on CI it failed so I reverted.

Reply:
> I have had it on my todo to get rid of the register_bigint<int64_t>("int64_t"); and unsigned version since that is actually not a "primitive" type. They're really long long int and unsigned long long int

I agree, just wasn't sure that I should do it in this PR. Might as well I guess.

Reply:
I think you need at least node 23 to run in 64 bit mode.

Reply:
I think node 24 ended up getting memory64 support.